### PR TITLE
Add Pinterest as option to social media channels

### PR DESCRIPTION
**Pull request by klickreflex**
Changes proposed in this pull request:

- Add option _Pinterest_ to social media globals
- Add Pinterest icon to resources 

I couldn't see any sort order within channels so I just appended it. 

(Personally I'd vote for alphabetic sorting because anything else will become only harder to grasp with a growing set of options, but that's beyond this PR)

### CODE REVIEW

**Comment by robdekort** [comment]
I have _just_ changed something in the Social Media globals so I'm gonna have to request a change. Also please do make it alphabetical. It was the intention at some point but clearly didn't stick ;-).

File: resources/blueprints/globals/social_media.yaml [hunk 1]
```diff
@@ -155,6 +155,24 @@ sections:
                   field:
                     type: hidden
                     default: 'https://www.github.com/'
+            Pinterest:
+              display: Pinterest
+              fields:
+                -
+                  handle: url
```
Comment (robdekort) [inline resources/blueprints/globals/social_media.yaml:162]: Can you change `url` to `handle`?

**Review by robdekort — CHANGES_REQUESTED** [review]

File: resources/blueprints/globals/social_media.yaml (hunk 1 above)
Comment (klickreflex) [reply resources/blueprints/globals/social_media.yaml:162]: Oh my I just saw I got the yaml messed up, will fix and sort

Comment (klickreflex) [reply resources/blueprints/globals/social_media.yaml]: Done. But it doesn't render correctly anymore. I can't see where I messed it up, can you @robdekort?

The handles aren't even saved it seems:
image

Comment (klickreflex) [reply resources/blueprints/globals/social_media.yaml]: Nevermind I've mixed old yaml of my current project with the new template. Should all be good here.

**Review by robdekort — APPROVED** [review]

File: resources/blueprints/globals/social_media.yaml (hunk 1 above)
Comment (robdekort) [reply resources/blueprints/globals/social_media.yaml]: Cool!

**Comment by robdekort** [comment]
Just noticed, please target the `/dev` folder next time.

**Pull request merged**